include/open62541/server.h

```diff
@@ -1553,6 +1553,71 @@ UA_Server_updateCertificate(UA_Server *server,
                             UA_Boolean closeSessions,
                             UA_Boolean closeSecureChannels);
 
+#ifdef UA_GENERATED_NAMESPACE_ZERO_FULL
+
+/* Set information on FileType Object instance
+ * 
+ * @param server The server object
+ * @param fileTypeObjectNodeId The NodeId of the fileType Object instance 
+ * @param fileName Name of the file created
+ * @param filePath Path to the file (TODO)
+ * @return UA_STATUSCODE_GOOD on success
+ */
+UA_StatusCode UA_EXPORT
```

Comment:
The callbacks you expose in the public API are much too detailed.
For the user it should be enough to have
```c
UA_Server_addFile(UA_Server *server, const UA_NodeId requestedNewNodeId,
                        const UA_NodeId parentNodeId,
                        const UA_NodeId referenceTypeId,
                        const UA_QualifiedName browseName,
                        const UA_ObjectAttributes attr,
                        const UA_String filePath, /* Load the file from here */
                        void *nodeContext, UA_NodeId *outNewNodeId);
```
All the methods and callbacks can be created and set internally from there and don't have to be exposed.

Reply:
Fixed it. Also added one more Public API `UA_Server_registerFileNode` to register the FileType object instance that added through information modeling.